{"version":1,"title":"Internal error in preempt_count_add (3)","id":"a18e03aa2c2f268f5e2f984d4cbd8338326a01ac","discussions":["http://b/419717218"],"crashes":[{"title":"Internal error in preempt_count_add","kernel-config":"/text?tag=KernelConfig\u0026x=201834d558a71127","kernel-source-git":"https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/log/?id=a11a72229881d8ac1d52ea727101bc9c744189c1","kernel-source-commit":"a11a72229881d8ac1d52ea727101bc9c744189c1","syzkaller-git":"https://github.com/google/syzkaller/commits/fa44301a1a289400e65667b9c05c597655d36157","syzkaller-commit":"fa44301a1a289400e65667b9c05c597655d36157","crash-report-link":"/text?tag=CrashReport\u0026x=1247f9f4580000"}]}